hack/testing/rsyslog/roles/rsyslog/tasks/main.yaml

```diff
@@ -0,0 +1,159 @@
+---
+- name: Install required packages
```

Comment:
Are the packages in the container image need to be installed or they are already installed and this should not be run when use_rsyslog_image is set to true?

Are the steps below relevant when we use the container image?